lib/src/slices/data_bkg/data_bkg_service.dart

```diff
@@ -208,4 +209,29 @@ class DataBkgService extends ChangeNotifier {
         "." +
         periodSplit[periodSplit.length - 1];
   }
+
+  String _gmailBuildQuery(int appDataGmailLastFetch, String category) {
+    StringBuffer queryBuffer = new StringBuffer();
+    if (appDataGmailLastFetch > 0) {
```

Comment:
if we're going to do a compare to a 0 then we should just pass in a nullable. otherwise use the 0